src/config.ts

```diff
@@ -7,5 +7,5 @@ export const dataPluginFolder = path.join(userDocuments, 'NI-Python-DataPlugins'
 export const extPrefix = 'NI DataPlugins: ';
 // eslint-disable-next-line operator-linebreak
 export const niConfig: vscode.WorkspaceConfiguration =
-    vscode.workspace.getConfiguration('NI-DataPlugins');
-export const exportPath: string | undefined = niConfig.get('PluginExportPath');
+    vscode.workspace.getConfiguration('DataPlugins');
```

Comment:
I think I would keep NI in the name of the config section here as kind of namespace either NI-DataPlugins.exportPath or even NI.DataPlugins.exportPath

Reply:
It seems to be uncommon to put the company name in the namespace.